quantum/quantum.c

```diff
@@ -451,7 +447,7 @@ bool process_record_quantum(keyrecord_t *record) {
         return false;
     }
 
-    return process_action_kb(record);
```

Comment:
The comment on lines 297 and 298 (just before this method) needs to be updated with this removal.

Reply:
I'm also wondering if the "This is how you use actions here" above is useful since it doesn't match the code at all.  It looks like advice from eons ago before some refactor.